.github/workflows/slack-notify.yml

```diff
@@ -0,0 +1,27 @@
+on:   
+  discussion:
+    types: [created]  
+name: Slack Notification Demo
+jobs:
+  slackNotification:
+    name: Slack Notification
+    runs-on: ubuntu-latest
+    steps:
+      - name: fetch the json value
+        run: |
+             githubjson=`cat $GITHUB_EVENT_PATH`
+             echo $githubjson
```

Comment:
Probably this echo was there for debugging purpose? If it's the case, we can remove it :)

Reply:
agreed! What's the best way to remove it?

Reply:
I think I figured out how to remove it